### ISSUE TITLE
Undefined function posix_getpwuid() for Eclipse IDE or Windows

### ISSUE THREAD

**Issue by shakaran** [googleads/googleads-php-lib#143]
Trying to load the OAuth2 credendials and running with Eclipse IDE in console:

    $oAuth2Credential = (new OAuth2TokenBuilder())
    ->fromFile()
    ->build(); 

I get this error, since in the php-cli for Eclipse IDE doesn't exist defined the php function posix_getpwuid:

     PHP Fatal error:  Call to undefined function posix_getpwuid() in somepath/vendor/googleads/googleads-php-lib/src/Google/AdsApi/Common/ConfigurationLoader.php on line 38

According to http://php.net/manual/es/function.posix-getpwuid.php#82387 in case that the function is not defined (checking with function_exists('posix_getpwuid') ) you can try getenv('USERNAME') or other way, but at least avoid the Fatal Error, even throw a exception. Also for windows php users that function will not be defined at all.

**Comment by shakaran** [comment]
I forgot to mention that it is experimental branch, (sorry I don't permissions to add Labels to the issue)

**Comment by vtsao** [comment]
Thanks for filing this, will look into it.

**Comment by vtsao** [comment]
Fixed in the latest version of experimental:
https://github.com/googleads/googleads-php-lib/tree/experimental

**Comment by shakaran** [comment]
Thanks @vtsao !! I can see a big diff commit, but it is hard to follow the changes between releases mostly because you are pushing moster commits. Could be possible that you commit in small commits per issue or changes related? I mean follow the standard principle for "Atomic Commits" https://en.wikipedia.org/wiki/Atomic_commit it would be a lot for see the progress
